upgrade/UPGRADE-v9.1.0.md

```diff
@@ -233,13 +332,34 @@ There you can find links to upgrade notes for other versions too.
     - `DomainFactoryOverwritingDomainUrl` is now replacing `DomainFactory` in `ACCEPTANCE` environment instead of `TEST`
 
 - add automatic string trimming and new Password type in frontend API ([#2127](https://github.com/shopsys/shopsys/pull/2127))
-    - see #project-base-diff to update your project
+    - see [project-base-diff](https://github.com/shopsys/project-base/commit/76dd04ed15d79208e0c66cd4bdf79f58c2e2020a) to update your project
 
 - change source of data for a single product to Elasticsearch in FE API ([#2131](https://github.com/shopsys/shopsys/pull/2131))
-    - see #project-base-diff to update your project
+    - see [project-base-diff](https://github.com/shopsys/project-base/commit/95d06cbddf63186ddee3a617292fa851587028bf) to update your project
     - run `php phing elasticsearch-index-migrate elasticsearch-export` to apply changes in Elasticsearch schema mapping
     - *BC BREAK* single product by UUID is now loaded from Elasticsearch with `productDetail` resolver
         - if necessary, you can switch to the former resolver `product` in your `Query.types.yaml` file
+    - following methods has changed their interface:
+        - `Shopsys\FrontendApiBundle\Model\Resolver\Price\PriceResolver::__construct()`
+            ```diff
+                public function __construct(
+                    ProductCachedAttributesFacade $productCachedAttributesFacade,
+                    ProductOnCurrentDomainFacadeInterface $productOnCurrentDomainFacade,
+                    PaymentPriceCalculation $paymentPriceCalculation,
+                    Domain $domain,
+                    CurrencyFacade $currencyFacade,
+            -       TransportPriceCalculation $transportPriceCalculation
+            +       TransportPriceCalculation $transportPriceCalculation,
+            +       ?PriceFacade $priceFacade = null
+                ) {
+            ```
+        - `Shopsys\ReadModelBundle\Product\Listed\ListedProductViewFactory`
+            ```diff
+            +   use Shopsys\FrameworkBundle\Model\Product\Pricing\PriceFactory;
+            -   use Shopsys\ReadModelBundle\Product\PriceFactory;
```

Comment:
I`m not sure whether this is helpful. 

The class should be an argument of the method or return type and in that case, I believe the exact method is more describing (and it would be a BC break :D)

If I see it right, it's used only in a new argument in the constructor, so I think it's not necessary to describe this change in upgrade notes as this move of the class is in the same version where it was added

Reply:
Yeah it was introduced in releasing version. I have removed it.